docs/source/filters.rst

```diff
@@ -143,9 +143,19 @@ There are several ways to get a list of possible keys for each resource.
 
       filters:
          - type: value
-           key: FunctionName                ─▶ The value from the describe call
+           key: FunctionName                ─▶ The value from the describe call, or resources.json
```

Comment:
I wondering if we should be saying key or attribute, since its not really a value, but given thats the previous ref here seems fine.

Reply:
Good point. Will add 'key' word before 'value' to be more specific, will that work?

Reply:
i'm fine with just leaving as is for now, we're going to need to try and do a whole doc narrative sweep at some point this year, let's just continue with things as is for now, if you want to followup with another doc pr, feel free to update this.